src/test/repository.test.ts

```diff
@@ -177,12 +183,15 @@ test('should request with etag', (t) =>
     repo.start();
   }));
 
-test('should request with custom headers', (t) =>
+test('should request with correct headers', (t) =>
```

Comment:
I see why you're co-opting the tests here, but we're also losing a bit of information about the tests with this rename. Before, it was clear that we're talking about custom headers, so if you're scanning through the tests, it was easy enough to find (or grep for). "Correct headers" is more ... ambiguous. How about renaming this one to something like this?
```suggestion
test('should request with custom and `x-unleash` headers', (t) =>
```

The same goes for the other tests that were changed from "custom" to "correct".

Reply:
Good call. I also updated one more test with the same change